mtp_api/apps/credit/views.py

```diff
@@ -112,10 +113,14 @@ class CreditListFilter(django_filters.FilterSet):
     user = django_filters.ModelChoiceFilter(name='owner', queryset=User.objects.all())
     received_at = CreditReceivedAtRangeFilter()
     search = CreditTextSearchFilter()
-    sender_name = django_filters.CharFilter(name='transaction__sender_name')
-    sender_sort_code = django_filters.CharFilter(name='transaction__sender_sort_code')
-    sender_account_number = django_filters.CharFilter(name='transaction__sender_account_number')
-    sender_roll_number = django_filters.CharFilter(name='transaction__sender_roll_number')
+    sender_name = django_filters.CharFilter(name='transaction__sender_name',
+                                            lookup_expr='icontains')
+    sender_sort_code = django_filters.CharFilter(name='transaction__sender_sort_code',
+                                                 lookup_expr='icontains')
+    sender_account_number = django_filters.CharFilter(name='transaction__sender_account_number',
+                                                      lookup_expr='icontains')
+    sender_roll_number = django_filters.CharFilter(name='transaction__sender_roll_number',
+                                                   lookup_expr='icontains')
```

Comment:
I don't think it's worth doing an ```icontains``` on the account details, and I think it will slow down DB queries if we index on these fields (which we may well).